packages/coqide/coqide.8.7.0/opam

```diff
@@ -4,23 +4,20 @@ version: "8.7.0"
 maintainer: "Maxime Dénès <mail@maximedenes.fr>"
 authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
 homepage: "https://coq.inria.fr/"
-bug-reports: "https://coq.inria.fr/bugs/"
+bug-reports: "https://github.com/coq/coq/issues"
 dev-repo: "https://github.com/coq/coq.git"
 license: "LGPL-2.1"
 build: [
   [
     "./configure"
-    "-configdir"
-    "%{lib}%/coq/config"
-    "-mandir"
-    man
-    "-docdir"
-    doc
-    "-prefix"
-    prefix
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
     "-usecamlp5"
-    "-camlp5dir"
-    "%{lib}%/camlp5"
+    "-camlp5dir" "%{lib}%/camlp5"
   ]
   [make "-j%{jobs}%" "coqide-files"]
   [make "-j%{jobs}%" "coqide-opt"]
```

Comment:
I think coqide also needs a constraint on ocaml-version < 4.06.0 due to not being safe-string compliant:

```
#=== ERROR while installing coqide.8.7.0 ======================================#
# opam-version 1.2.2
# os           linux
# command      make -j4 coqide-opt
# path         /home/travis/.opam/4.06.0/build/coqide.8.7.0
# compiler     4.06.0
# exit-code    2
# env-file     /home/travis/.opam/4.06.0/build/coqide.8.7.0/coqide-24630-384e1f.env
# stdout-file  /home/travis/.opam/4.06.0/build/coqide.8.7.0/coqide-24630-384e1f.out
# stderr-file  /home/travis/.opam/4.06.0/build/coqide.8.7.0/coqide-24630-384e1f.err
### stdout ###
# [...]
# OCAMLC    vernac/auto_ind_decl.mli
# OCAMLOPT  ide/ideutils.ml
# OCAMLOPT  ide/coqide_ui.ml
# OCAMLOPT  kernel/context.ml
# OCAMLC    engine/ftactic.mli
# OCAMLC    pretyping/evarsolve.mli
# OCAMLC    pretyping/find_subterm.mli
# OCAMLC    proofs/proof_type.mli
# OCAMLC    engine/geninterp.mli
# make[1]: Leaving directory `/home/travis/.opam/4.06.0/build/coqide.8.7.0'
### stderr ###
# File "ide/ideutils.ml", line 376, characters 38-74:
# Error: This expression has type string but an expression was expected of type
#          bytes
# make[1]: *** [ide/ideutils.cmx] Error 2
# make[1]: *** Waiting for unfinished jobs....
# make: *** [submake] Error 2
```

Reply:
Ah, right. The upcoming 8.7.1 will fix that.